rts/Sim/Units/Unit.cpp

```diff
@@ -246,6 +246,16 @@ void CUnit::PreInit(const UnitLoadParams& params)
 
 	SetVelocity(params.speed);
 	Move(preFramePos = params.pos.cClampInMap(), false);
+
+	// Force buildings to the 2x2 grid. This normally enforced, but buildings spawned by maps have not
+	// been forced yet.
```

Comment:
Is "the 2x2 grid" the one where a single cell consists of 2x2 of the smaller ones, or the one where 2x2 single cells fit within one larger one?

Also maps cannot spawn buildings on their own but Lua and cheats can.
```suggestion
	// Force buildings to the 16x16 elmo grid (normally done for construction but not Lua or /give)
```